eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -95,14 +107,24 @@ impl Eth2NearRelay {
         while !self.terminate {
             iter_id += 1;
             self.set_terminate(iter_id, max_iterations);
+            skip_fail!(self.wait_for_synchronization(), "Fail to get sync status");
 
             info!(target: "relay", "== New relay loop ==");
+            sleep(Duration::from_secs(12));
```

Comment:
We should avoid sleeping in two cases:
1. on the first run
2. when there is big gap, so we want to sync it as fast as possible

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/800/commits/3b32c83caba171589438821e002e00ef14c9d08e